mle/cli.py

```diff
@@ -1,6 +1,9 @@
 import os
 import yaml
 import click
+import pickle
+import calendar
```

Comment:
Seems the calendar and datetime are unused

Reply:
yes, I will remove these unused modules :)

Reply:
I can help you to do this in my PR, no worries : )